CTP/shell/src/com/navercorp/cubridqa/shell/result/FeedbackFile.java

```diff
@@ -84,30 +98,76 @@ public void onTaskContinueEvent() {
 
 	@Override
 	public void onTaskStopEvent() {
+		showTestResult();
+		
 		long taskStopTime = System.currentTimeMillis();
 		println("[TEST STOP] Current Time is " + new Date(), "Elapse Time:" + ((taskStopTime - this.taskStartTime)));
 		feedbackLog.close();
 	}
 
 	@Override
 	public void setTotalTestCase(int tbdNum, int macroSkippedNum, int tempSkippedNum) {
+		this.totalCaseNum = tbdNum;
+		this.totalSkipNum = macroSkippedNum + tempSkippedNum;
 		println("The Number of Test Cases: " + tbdNum + " (macro skipped: " + macroSkippedNum + ", bug skipped: " + tempSkippedNum + ")");
 	}
+	
+	public void showTestResult(){
+		System.out.println("============= PRINT SUMMARY ==================");
```

Comment:
As is:
System.out.println("============= PRINT SUMMARY ==================");

To be:
println("============= PRINT SUMMARY ==================");

And also change others.